meinberlin/apps/plans/migrations/0062_alter_plan_cost_alter_plan_description_and_more.py

```diff
@@ -0,0 +1,117 @@
+            name="cost",
+            field=models.CharField(max_length=255, verbose_name="Cost"),
+        ),
+        migrations.AlterField(
+            model_name="plan",
+            name="description",
+            field=django_ckeditor_5.fields.CKEditor5Field(
+                verbose_name="Description of your plan"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="plan",
+            name="district",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                to="a4administrative_districts.administrativedistrict",
+                verbose_name="District",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="plan",
+            name="duration",
+            field=models.CharField(
+                blank=True, max_length=255, null=True, verbose_name="Duration"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="plan",
+            name="image_copyright",
+            field=models.CharField(
+                blank=True, max_length=120, verbose_name="Header image copyright"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="plan",
+            name="participation",
+            field=models.SmallIntegerField(
+                choices=[
+                    (0, "information (no participation)"),
+                    (1, "consultation"),
+                    (2, "cooperation"),
+                    (3, "decision-making"),
+                ],
+                default=0,
+                verbose_name="Level of participation",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="plan",
+            name="participation_explanation",
+            field=models.TextField(
+                max_length=4000, verbose_name="Participation explanation"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="plan",
+            name="point",
+            field=adhocracy4.maps.fields.PointField(
+                blank=True, verbose_name="Can your plan be located on the map?"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="plan",
+            name="point_label",
+            field=models.CharField(
+                blank=True, default="", max_length=255, verbose_name="Name of the site"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="plan",
+            name="status",
+            field=models.SmallIntegerField(
+                choices=[(0, "running"), (1, "done")], verbose_name="Status"
+            ),
+        ),
+        migrations.AlterField(
+            model_name="plan",
+            name="tile_image_copyright",
+            field=adhocracy4.images.fields.ImageCopyrightField(
+                blank=True,
+                help_text="Copyright shown in the image.",
```

Comment:
hmm, why here is added? u are removing it in the model. Is this coming from the mixin?

Reply:
ah :/ yes, that's from a4, I didn't touch that